static/js/unread.js

```diff
@@ -264,9 +264,55 @@ exports.num_unread_for_person = function (user_ids_string) {
     return unread_privates.get(user_ids_string).num_items();
 };
 
+exports.current_stream_topic = function () {
+    var selected_stream;
+    var selected_topic;
+    if (narrow.filter() === undefined || narrow.filter()._operators["1"] === undefined) {
+         var message_header = $(".selected_message").closest(".recipient_row").find(".message_header");
```

Comment:
I tried to use minimal amount of JQuery. The ` narrow.filter()` function doesn't always return the stream and topic values. As far as I tried these work when the user is home view.